templates/repo/settings/push_mirror_sync_modal.tmpl

```diff
@@ -2,31 +2,27 @@
 	<div class="header">
 		{{ctx.Locale.Tr "repo.settings.mirror_settings.push_mirror.edit_sync_time"}}
 	</div>
-	<div class="content">
-		<form class="ui form ignore-dirty" method="post">
-			{{.CsrfTokenHtml}}
-			<input type="hidden" name="action" value="push-mirror-update">
-			<input type="hidden" name="push_mirror_id" id="push-mirror-edit-id">
-			<div class="field">
-				<label for="name">{{ctx.Locale.Tr "repo.settings.mirror_settings.mirrored_repository"}}</label>
-				<div class="ui small input">
-					<input id="push-mirror-edit-address" readonly>
```

Comment:
This breaks changing the mirror URL.
At the moment (1.23.4), it is possible to change it.

Reply:
The related PRs are:

* #26151
* #27487

The URL is not changeable.

Reply:
Yep, you are right, I got completely bamboozled:
I saw the `readonly` in the code above, but didn't check in the actual textbox if I could edit it.
The textbox did not look disabled, so I assumed it could be edited…
I thought this may have been handled with JS.
I didn't imagine in the slightest that we would have a read-only input instead of a span that does not differ at all from an editable input.